evm_loader/program/config/common.toml

```diff
@@ -0,0 +1,18 @@
+[u64]
```

Comment:
Can we get the type from the value?
We can work only with 2 types of values: integer (as u64) and string.

Reply:
It's doable.